aws-encryption-sdk-cpp/CMakeLists.txt

```
@@ -113,6 +113,24 @@ if (AWS_ENC_SDK_KNOWN_GOOD_TESTS)
         ${PROJECT_SOURCE_DIR}/tests/integration
         $<INSTALL_INTERFACE:include>
         )
+
+    # Also unzip the test vectors from the aws-encryption-sdk-test-vectors submodule
+    # TODO-RS: If desired, we can use a custom target to avoid doing this
+    # on every build when the zip hasn't changed.
+    message(STATUS "Unzipping test vectors")
+    set(TEST_VECTORS_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/test-vectors)
+    file(MAKE_DIRECTORY ${TEST_VECTORS_DIRECTORY})
+    execute_process(COMMAND ${CMAKE_COMMAND} -E tar xvf ${AWS_ENC_SDK_TEST_VECTORS_ZIP}
+        WORKING_DIRECTORY ${TEST_VECTORS_DIRECTORY}
+        RESULT_VARIABLE _exit_code
+        # The output from the subprocess seems to fill a buffer and cause cmake
+        # to crash on Ubuntu, so don't remove this!
+        OUTPUT_QUIET)
```

nit: Could we avoid needing `OUTPUT_QUIET` if we omit the `v` flag to `tar`? i.e.

```suggestion
    execute_process(COMMAND ${CMAKE_COMMAND} -E tar xf ${AWS_ENC_SDK_TEST_VECTORS_ZIP}
        WORKING_DIRECTORY ${TEST_VECTORS_DIRECTORY}
        RESULT_VARIABLE _exit_code)
```

---

Ah good callout, assuming this passes on CodeBuild I'm much happier with this.